htdocs/core/class/evalmath.class.php

```diff
@@ -374,7 +374,7 @@ private function nfx($expr)
 	 */
 	private function pfx($tokens, $vars = array())
 	{
-		if ($tokens == false) {
+		if (empty($tokens)) {
```

Comment:
Because you change code of an utility function, you must also add a phpunit test for the class
EvalMath.